src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/k8s-version2/troubleshooting.mdx

```diff
@@ -0,0 +1,294 @@
+       ```
+
+       As a workaround, increase the DaemonSet memory limits so the process is not killed.
+  </Collapser>
+
+  <Collapser
+    id="cannot-list-pods-for-cluster"
+    title="Cannot list pods at the cluster scope"
+  >
+    Newrelic pods and newrelic service account are not deployed in the same namespace. This is usually because the current context specifies a namespace. If this is the case, you will see an error like the following:
+
+    ```shell
+    time=\"2018-05-31T10:55:39Z\" level=panic msg=\"p
+    ods is forbidden: User \\\"system:serviceaccount:kube-system:newrelic\\\" cannot list pods at the cluster scope\"
+    ```
+
+    To check to see if this is the case, run:
+
+    ```shell
+    kubectl describe serviceaccount newrelic | grep Namespace
+    kubectl get pods -l name=newrelic-infra --all-namespaces
+    kubectl config get-contexts
+    ```
+
+    To resolve this problem, change the namespace for the service account in the New Relic `DaemonSet` YAML file to be the same as the namespace for the current context:
+
+    ```yaml
+    - kind: ServiceAccount
+      name: newrelic
+      namespace: default
+    ---
+    ```
+  </Collapser>
+</CollapserGroup>
+
+
+## Not seeing control plane data [#control-plane-data]
+
+<CollapserGroup>
+  <Collapser
+    id="invalid-license"
+    title="Check that the master nodes have the correct labels"
+  >
+    Execute the following commands to manually find the master nodes:
+
+    ```shell
+    kubectl get nodes -l node-role.kubernetes.io/master=""
+    ```
+
+    ```shell
+    kubectl get nodes -l kubernetes.io/role="master"
+    ```
+
+    If the master nodes follow the labeling convention defined in the [discovery of master nodes and control plane components documentation section](/docs/integrations/kubernetes-integration/installation/configure-control-plane-monitoring#discover-nodes-components), you should get some output like:
+
+    ```shell
+    NAME                         STATUS  ROLES   AGE   VERSION
+    ip-10-42-24-4.ec2.internal   Ready   master  42d   v1.14.8
+    ```
+
+    If no nodes are found, there are two scenarios:
+
+    Your master nodes don't have the required labels that identify them as masters, in this case you need to add both labels to your master nodes.
+
+    You're in a managed cluster and your provider is handling the master nodes for you. In this case there is nothing you can do, since your provider is limiting the access to those nodes.
+  </Collapser>
+
+  <Collapser
+    id="unable-connect"
+    title="Check that the integration is running on the master nodes"
+  >
+    Replace the placeholder in the following command with one of the node names returned in the previous step to get an integration pod running on a master node:
+
+    ```shell
+    kubectl get pods --field-selector spec.nodeName=NODE_NAME -l name=newrelic-infra --all-namespaces
+    ```
+
+    The next command is the same, just that it selects the node for you:
```

Comment:
This phrasing is confusing. Should this be an either/or with the previous command, like you can use one of the following commands depending on your needs?